rmgpy/data/solvation.py

```diff
@@ -266,6 +266,79 @@ def average_solute_data(solute_data_list):
             averaged_solute_data.A /= num_values
             return averaged_solute_data
 
+
+def get_critical_temperature(compound_name):
+    """Returns the critical temperature of a compound in K for the given compound_name.
+
+    The critical temperature is given by the CoolProp function, PropsSI
+
+    Args:
+        compound_name (str): a name of the compound used in CoolProp.
+
+    Returns:
+        Tc (float): critical temperature of the given compound in K.
+
+    Raises:
+        DatabaseError: If the given compound_name is not available in CoolProp.
+
+    """
+
+    try:
+        Tc = PropsSI('T_critical', compound_name)
+    except:
+        raise DatabaseError(f"Critical temperature is not available for the given compound: {compound_name}")
+    return Tc
+
+
+def get_liquid_saturation_density(compound_name, temp):
+    """Returns the liquid-phase saturation density of a compound in mol/m^3 at an input temperature.
+
+    The density is calculated by the CoolProp function, PropsSI.
+
+    Args:
+        compound_name (str): a name of the compound used in CoolProp.
+        temp (float): temperature at which the liquid-phase saturation density is calculated. Must it in Kelvin.
+
+    Returns:
+        rho_l (float): liquid-phase saturation density at the given temperature in mol/m^3.
+
+    Raises:
+        DatabaseError: If the given compound_name is not available in CoolProp or the given temperature is out of
+        the calculable range.
+
+    """
+
+    try:
+        rho_l = PropsSI('Dmolar', 'T', temp, 'Q', 0, compound_name)  # saturated liquid phase density in mol/m^3
+    except:
```

Comment:
Same here

Reply:
I found out that more than one type of exception could be thrown by PropsSI function, so I decided not to specify the type of exception here. (same answer as above)